starter-code/js/index.js

```diff
@@ -11,7 +11,26 @@ function updatePriceByProduct(productPrice, index){
 }
 
 function getTotalPrice() {
-
+  
+  var price = document.querySelectorAll(".product-price");
+  var quatity = document.querySelectorAll(".quantity");  
```

Comment:
Defining those variables with querySelectorAll so they're arrays of values is good, so you can loop through them.